cmd/setup-etcd-environment/run.go

```diff
@@ -27,6 +27,10 @@ import (
 const (
 	EtcdScalingAnnotationKey = "etcd.operator.openshift.io/scale"
 	etcdInitialExisting      = "existing"
+	retryDuration            = 10 * time.Second
+	testIPAddress            = "127.0.0.256"
```

Comment:
can we have test-related constants in the test file to unclutter this?

Reply:
sure